machine-learning/Intro To Machine Learning.ipynb

```diff
@@ -0,0 +1,320 @@
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Using the `MLContext` we previously created, load the training data into ML.NET [IDataView](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.idataview?view=ml-dotnet) which is the fundamental ML.NET data pipeline type"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "IDataView trainingData = mlContext.Data.LoadFromEnumerable(houseData);"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now we have the data ready, next we'll create the ML.NET pipeline specifying the training model we are going to use for our ML operation. For house price prediction, we are going to use the Regression transformer model. ML.NET supports other machine learning models which can be used for other scenarios as needed. The pipeline will create what is called [Estimator](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.iestimator-1?view=ml-dotnet) which used for training the data"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "// 2. Specify data preparation and model training pipeline\n",
+        "var pipeline = mlContext.Transforms.Concatenate(\"Features\", new[] { \"Size\" })\n",
+        "               .Append(mlContext.Regression.Trainers.Sdca(labelColumnName: \"Price\", maximumNumberOfIterations: 100));\n",
+        ""
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "After creating the estimator, we are ready to train the data using the regression transformer model. To do that, call the [Fit](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.iestimator-1.fit?ranMID=43674&ranEAID=rl2xnKiLcHs&ranSiteID=rl2xnKiLcHs-G8Db905fJ0jxggGna1mdkw&epi=rl2xnKiLcHs-G8Db905fJ0jxggGna1mdkw&irgwc=1&OCID=AID2200057_aff_7795_1243925&tduid=(ir__2m3q0nl02wkf6gcatnnkkvci0e2xvx1gft3xgf9200)(7795)(1243925)(rl2xnKiLcHs-G8Db905fJ0jxggGna1mdkw)()&irclickid=_2m3q0nl02wkf6gcatnnkkvci0e2xvx1gft3xgf9200&view=ml-dotnet) method."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "           var model = pipeline.Fit(trainingData);"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now we have the trained model ready which can be used for house price prediction. Let's use this model to predict a sample house price."
```

Comment:
```suggestion
        "Now we have the trained model which can be used for house price prediction. Let's use this model to predict a sample house price."
```

Reply:
This would also be a good opportunity to introduce and describe `PredictionEngine`

Reply:
I would also recommend before making predictions, adding a section on evaluation especially since it's shown in the diagram below.